src/transaction.ts

```diff
@@ -92,6 +91,11 @@ export class Transaction {
      */
     public guardian: string;
 
+    /**
+     * The relayer, in address format, next version all the other addresses will not be string anymore.
+     */
+    public relayer?: Address;
```

Comment:
Indeed, optional - I see we have the same in PY :+1: 

https://github.com/multiversx/mx-sdk-py/blob/feat/next/multiversx_sdk/core/transaction.py#L49